src/components/tests/Quote.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Kindly avoid using comments to skip linter errors, as it is not a good practice. Sometimes linter error happens due to syntax error or anything related to the code. So try to fix it manually and this way you may learn a new concept.